xml/admin_saltcluster.xml

```diff
@@ -1168,6 +1168,18 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
      actual replacement, but will preview the steps that would normally happen.
     </para>
    </tip>
+   <important>
+     <note>
+       <para>
+         When replacing an OSD, we highly recommend triggering a deep scrub of
+         the placement groups. See <xref linkend="scrubbing-pgs"/> for more details.
```

Comment:
Probably worth mentioning that this is for the case when replacing after osd failure. If there was some other reason for replacement they will probably do not need to deep-scrub.